opa/Dockerfile

```diff
@@ -85,32 +85,41 @@ FROM stackable/image/vector
 
 ARG PRODUCT
 ARG RELEASE
+ARG STACKABLE_USER_UID
 
 LABEL name="Open Policy Agent" \
       maintainer="info@stackable.tech" \
       vendor="Stackable GmbH" \
       version="${PRODUCT}" \
       release="${RELEASE}" \
-      summary="The Stackable image for OPA." \
+      summary="The Stackable image for Open Policy Agent (OPA)." \
       description="This image is deployed by the Stackable Operator for OPA."
 
-RUN microdnf update && \
-    microdnf install \
-    # Required for filtering logs
-    jq && \
-    microdnf clean all && \
-    rm -rf /var/cache/yum
-
 COPY opa/licenses /licenses
 
-USER stackable
-WORKDIR /stackable/opa
+COPY --from=opa-builder --chown=${STACKABLE_USER_UID}:0 /opa/opa /stackable/opa/opa
+COPY --from=opa-bundle-builder --chown=${STACKABLE_USER_UID}:0 /opa-bundle-builder/target/release/stackable-opa-bundle-builder /stackable/opa-bundle-builder
+COPY --from=multilog-builder --chown=${STACKABLE_USER_UID}:0 /daemontools/admin/daemontools/command/multilog /stackable/multilog
 
-COPY --from=opa-builder /opa/opa /stackable/opa/opa
-COPY --from=opa-bundle-builder --chown=stackable:stackable /opa-bundle-builder/target/release/stackable-opa-bundle-builder /stackable/opa-bundle-builder
-COPY --from=multilog-builder --chown=stackable:stackable /daemontools/admin/daemontools/command/multilog /stackable/multilog
+COPY --chown=${STACKABLE_USER_UID}:0 opa/stackable/bin /stackable/opa/bin
 
-COPY --chown=stackable:stackable opa/stackable/bin /stackable/opa/bin
+RUN <<EOF
+microdnf update
+
+# jq: Required for filtering logs
+microdnf install \
+  jq
```

Comment:
Just noting that we install jq in the tools image. I'm unsure where that is used.

Reply:
I'm not sure I understand. Can you explain that to me? Is the tools image used here?

Reply:
What is the intention of the tools image? I assumed it was so we could use `COPY` to pull out tools for use, rather than installing them via microdnf. But maybe it is for a totally different use case.

Or, if we install `jq` via microdnf, why don't we do the same for tools?

https://github.com/stackabletech/docker-images/blob/d3231ba9d6291c7c1f31404e3e2c472d433f3042/tools/Dockerfile#L41-L43

Reply:
Ah.... I don't actually know....sorry.

Reply:
No problem. I will find out separately from this PR.